demo/example.py

```diff
@@ -13,38 +13,38 @@
 
 @app.route("/")
 def hello():
-    return "SSRF Example!"
+    return "Simple SSRF Demo App"
 
 
-# curl -i -X POST -d 'url=http://example.com' http://localhost:5000/ssrf
-@app.route("/ssrf", methods=["POST"])
-def ssrf():
+# curl -i -X POST -d 'url=http://example.com' http://localhost:5000/http-post-form-data
```

Comment:
`http` and `post` seem redundant here. `form-data` is good enough IMO. Same for below endpoints too.